src/main/java/com/bootcamp/demo/service/assembler/ScooterAssembler.java

```diff
@@ -0,0 +1,11 @@
+package com.bootcamp.demo.service.assembler;
+
+import com.bootcamp.demo.model.Scooter;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+
+public class ScooterAssembler {
```

Comment:
This class no longer exists in master. Take care not to add it, since it is not used and as such, it is not needed.